src/app/config.rs

```diff
@@ -611,6 +630,11 @@ impl<'a> TryFrom<&'a Strings> for Config {
             dns_max_ttl: dns_max_ttl?,
 
             dns_canonicalize_timeout,
+
+            h2_settings: H2Config {
```

Comment:
Pretty minor nit, but I don't like that the field is named `h2_settings` but the type is `h2_config` (this pattern shows up elsewhere in the PR as well). Can we call both either `H2Settings` or `h2_config`?

Reply:
I agree this looks awkward. I like `H2Settings` because it reflects that that these values are HTTP/2 settings that we are overriding: https://http2.github.io/http2-spec/#SETTINGS_INITIAL_WINDOW_SIZE